test/test_prototype_transforms.py

```diff
@@ -18,7 +18,7 @@
 )
 from torchvision.ops.boxes import box_iou
 from torchvision.prototype import features, transforms
-from torchvision.transforms.functional import InterpolationMode, pil_to_tensor, to_pil_image
+from torchvision.transforms.functional import get_image_num_channels, InterpolationMode, pil_to_tensor, to_pil_image
```

Comment:
We shouldn't be using `get_image_num_channels` kernels from `torchvision.transforms.functional` here. Instead we should only be using from the prototype area.